modules/ossm-migrating-premigration-checklists-using-cert-manager.adoc

```diff
@@ -0,0 +1,38 @@
+// Module included in the following assemblies:
+//
+// * service-mesh-docs-main/migrating/checklists/ossm-migrating-premigration-checklists-assembly.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="ossm-migrating-premigration-checklists-using-cert-manager_{context}"]
+= Check if you are using the cert-manager tool with your deployment
```

Comment:
Please see the mod docs guide on how to format procedure titles. For consistency, it would be best to reformat your filename and id to match the updated title.